app/api_topology_test.go

```diff
@@ -112,6 +91,7 @@ func TestAPITopologyHosts(t *testing.T) {
 	defer ts.Close()
 	is404(t, ts, "/api/topology/hosts/foobar")
 	{
+		render.ResetCache() // TODO figure out why this is needed.
```

Comment:
Why *is* this needed? each report should have a different ID, yeah? ID collisions in the renderers, or reports?